extensions/webdav/pom.xml

```diff
@@ -52,6 +52,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>commons-beanutils</groupId>
```

Comment:
Perhaps add a comment saying why this is hear - as it is transient, it typically should not be here - which means that without extra information, someone could easily remove this in future by mistake.

Reply:
@adamretter yes makes sense